application/helpers/common_helper.php

```
@@ -4200,6 +4200,118 @@ function replaceExpressionCodes($iSurveyID, $aCodeMap)
+        foreach ($arQuestion->conditions as $condition) {
+            $cfieldname = $condition->cfieldname;
+            foreach ($aQIDReplacements as $sOldFieldname => $sNewFieldname) {
+                $cfieldname = replaceFieldnameMatches($cfieldname, $sOldFieldname, $sNewFieldname);
+            }
+            if ($condition->cfieldname !== $cfieldname) {
+                $condition->cfieldname = $cfieldname;
+                $condition->save();
+            }
+        }
+        foreach ($arQuestion->questionl10ns as $arQuestionLS) {
+            $bModified = false;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$bModified'. (undefined)

(UnusedLocalVariable)